src/app/static/src/app/components/projectListPage.vue

```diff
@@ -1,22 +1,38 @@
 <template>
-     <div class="container mt-5">
+    <div class="container mt-5">
         <div class="row">
             <div class="col-lg-6 offset-lg-3 col-10 offset-1">
+                <p>
+                    This tool is designed to help National Malaria Control Programs explore the most cost effective
```

Comment:
I wonder if this should be collapsible too, and collapsed by default if the user has at least one project i.e. we can assume they've already seen it. It feels a little intrusive having this here when you want to be editing projects..

Reply:
I dunno, we have a lot of collapsible content now, feels like this is ok as contextual info about projects/regions to display all the time? I'm minded to leave as is unless anything is raised by others in user testing?